components/chainhook-sdk/src/indexer/mod.rs

```diff
@@ -22,45 +25,6 @@ pub struct AssetClassCache {
     pub decimals: u8,
 }
 
-#[derive(Deserialize, Debug, Clone)]
-pub struct PoxConfig {
-    pub first_burnchain_block_height: u32,
-    pub prepare_phase_block_length: u32,
-    pub reward_phase_block_length: u32,
-}
-
-impl PoxConfig {
-    pub fn mainnet_default() -> PoxConfig {
-        PoxConfig {
-            first_burnchain_block_height: 666050,
-            prepare_phase_block_length: 100,
-            reward_phase_block_length: 2000,
```

Comment:
@hugocaillard The `reward_phase_block_length` that we are removing here differs slightly from what we're keeping above. The same goes for devnet/testnet. Is this at all cause for concern? Do you know which numbers are correct?

Reply:
@MicaiahReid 
Yeah I think it a concern.

Ideally we should have `reward_length + prepare_length = cycle_length`
But, in some places in clarinet, stacks-core, and chainhook, we have a wrong wording for that in way that
`cycle_length = reward_length`, so `actual_reward_length = reward_length - prepare_length`

Soooo, yeah, it's kind of a mess, and just here in chainhook, we have the issue, with a prepare phase of 100, and a reward phase of 2000 or 2100. 

But the true reward phase on mainnet is 2000, so the cycle length is 2100.

If we have places were we use it, we probably want adjust the maths

Reply:
@tippenein, do you think that's enough info to work with? Maybe we can all get on a call to make sure it's clear how to approach this

Reply:
And this is great: https://github.com/hirosystems/chainhook/blob/pox-config-refactor-547/components/chainhook-sdk/src/chainhooks/types.rs#L591-L593